helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/collectiontransformer/StringListView.java

```diff
@@ -38,16 +38,30 @@ public String set(int index, String element) {
     return previousValue;
   }
 
+  public String set(int index, Utf8 element) {
+    String previousValue = String.valueOf(_utf8List.get(index));
+    _utf8List.set(index, element);
+    return previousValue;
+  }
+
   @Override
   public void add(int index, String element) {
     _utf8List.add(index, new Utf8(element));
   }
 
+  public void add(int index, Utf8 element) {
+    _utf8List.add(index, element);
+  }
+
```

Comment:
no such method in the base class?

Reply:
No. Base class only has methods for String in this case.. (Since we are extending AbstractList<String>).